tests/sdk-consumer/app.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once __DIR__ . '/vendor/autoload.php';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ["require_once" statement detected. File manipulations are discouraged. Concatenating is forbidden.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)